README.md

```diff
@@ -201,23 +205,38 @@ image. The numpy object should be in int8, [Y,X,RGB] format.
 A minimal example might be:
 
 ```
+from tflite_runtime.interpreter import Interpreter
+import os
 import numpy as np
 from PIL import Image
+from PIL import ImageDraw
 from pose_engine import PoseEngine
 
-pil_image = Image.open('couple.jpg')
-pil_image.resize((641, 481), Image.NEAREST)
 
-engine = PoseEngine('models/mobilenet/posenet_mobilenet_v1_075_481_641_quant_decoder_edgetpu.tflite')
-poses, inference_time = engine.DetectPosesInImage(np.uint8(pil_image))
-print('Inference time: %.fms'%inference_time)
+os.system('wget https://upload.wikimedia.org/wikipedia/commons/thumb/3/38/'
+          'Hindu_marriage_ceremony_offering.jpg/'
+          '640px-Hindu_marriage_ceremony_offering.jpg -O /tmp/couple.jpg')
+
+pil_image = Image.open('/tmp/couple.jpg')
+engine = PoseEngine(
+    'models/mobilenet/posenet_mobilenet_v1_075_481_641_quant_decoder_edgetpu.tflite')
+poses, inference_time = engine.DetectPosesInImage(pil_image)
+print('Inference time: %.fms' % inference_time)
 
+draw = ImageDraw.Draw(pil_image)
```

Comment:
I don't think image draw is necessary for a minimal example. Can it be left as it is?